src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/SemanticTokensTest.cs

```diff
@@ -22,6 +22,7 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
 using Microsoft.CodeAnalysis.Razor.Workspaces;
+using Microsoft.CodeAnalysis.Razor.Workspaces.ProjectSystem;
```

Comment:
Just by way of example, if you change the namespaces of the types you moved to Workspaces to not include "Workspaces", this file won't need to be updated.